docs/sdk/api/graphQLTypescriptSdk.md

```diff
@@ -0,0 +1,402 @@
+```js
+  const { createClient } = require('@commercetools/sdk-client')
+  const { createAuthMiddlewareForClientCredentialsFlow } = require('@commercetools/sdk-middleware-auth')
+  const { createHttpMiddleware } = require('@commercetools/sdk-middleware-http')
+  const { createApiBuilderFromCtpClient } = require("@commercetools/typescript-sdk")
+  const fetch = require('node-fetch')
+  require('dotenv').config()
+  //reference API client credentials from environment variables
+  const {
+    CTP_PROJECT_KEY,
+    CTP_CLIENT_SECRET,
+    CTP_CLIENT_ID,
+    CTP_AUTH_URL,
+    CTP_API_URL,
+    CTP_SCOPES,
+  } = process.env
+  const projectKey = CTP_PROJECT_KEY
+  const authMiddleware = createAuthMiddlewareForClientCredentialsFlow({
+    host: CTP_AUTH_URL,
+    projectKey,
+    credentials: {
+      clientId: CTP_CLIENT_ID,
+      clientSecret: CTP_CLIENT_SECRET,
+    },
+    scopes: [CTP_SCOPES],    fetch,
+  })
+  const httpMiddleware = createHttpMiddleware({
+    host: CTP_API_URL,
+    fetch,
+  })
+  const client = createClient({
+    middlewares: [authMiddleware, httpMiddleware],
+  })
+  // Create an API root from API builder of commercetools platform client
+  const apiRoot = createApiBuilderFromCtpClient(client);
+  console.log('Getting started with commercetools Typescript SDK and GraphQL API');
+```
+This code creates a **client**, which uses `authMiddleware` and `httpMiddleware`. The `httpMiddleware` reads the `clientId` and `clientSecret` from environment variables. 
+
+Install the `@commercetools/sdk-client`, `@commercetools/sdk-middleware-auth`, `@commercetools/sdk-middleware-http`, `@commercetools/typescript-sdk` and `dotenv` packages and save it to your `package.json` dependencies using the following command:
+
+```
+$ npm install @commercetools/sdk-client @commercetools/sdk-middleware-auth @commercetools/sdk-middleware-http @commercetools/typescript-sdk dotenv
+```
+
+Run the program using the following command:
+```
+$ node project.js
+Getting started with commercetools Typescript SDK and GraphQL API
+```
+If you see the same output as above, we’re ready to start.
+
+### Get commercetools project settings with the commercetools GraphQL API
+In this guide, we’ll get project settings information using commercetools GraphQL API. 
+
+Re-open `project.js` and add the following code:
+```js
+// GraphQL query to get commercetools project settings
+const projectSettingsQuery = `
+    query {
+        project {
+        name
+        languages
+        currencies
+        countries
+        version
+        createdAt
+        }
+    }
+`;
+
+(async () => {
+    try {
+       await apiRoot.withProjectKey({projectKey}).graphql()
+            .post({
+                body : {
+                    query: projectSettingsQuery,
+                    variables: {}
+                }
+            })
+            .execute()
+            .then(data => {
+                console.log('Project information --->', data);
```

Comment:
This gives me following output:
`Project information ---> { body: { data: { project: [Object] } }, statusCode: 200 }`
Is there a way to look deeper into the `Object`, like:
```suggestion
                console.log('Project information --->', data.body);
```

Reply:
@stmeissner We might have to use `JSON.stringify(data)`.